CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ unset(${SPECTRE_NAME}_VERSION_PATCH)
 if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.12)
   cmake_policy(SET CMP0074 NEW)
 endif ()
+# - We use test names with '.' characters in them
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.19)
+  cmake_policy(SET CMP0110 NEW)
```

Comment:
Would you mind deleting the lines 30-32 above? We require cmake 3.12 now, so the policy is set to `NEW` by default.

Reply:
Certainly, I'll update the PR 🙂

Reply:
I think we can actually delete most (all) of the `<Package>_ROOT` variables now, since cmake uses them by default. We only have be careful about capitalization, i.e. we'll have to update the relevant documentation to use `Blaze_ROOT` instead of `BLAZE_ROOT`. The capitalization follows the spelling of the corresponding `Find<Package>.cmake` file, so we can also adjust that in some cases if we want. Not very high priority, but it's a possible cleanup now. I might do it at some point, but feel free to do it as well (please let me know if you do so we don't duplicate work).

Reply:
Sounds good. I have no immediate plans to do this, but I'll check with you before starting it.